test-cypress/steps/setup/user-and-access-steps.js

```diff
@@ -205,4 +205,125 @@ export class UserAndAccessSteps {
             // Force the click because Cypress sometimes determines that the item has 0x0 dimensions
             .click({force: true});
     }
+
+    static clickGraphqlAccessForRepo(repoName) {
```

Comment:
Why do you complicate the steps? There should be no conditional logic in the steps and in the tests.

Reply:
fixed